serviceobjects/src/main/java/PassengerSO.java

```diff
@@ -0,0 +1,75 @@
+import java.sql.Date;
+import java.util.Collection;
+
+public class PassengerSO {
+
+    private int id;
+    private String firstName;
+    private String lastName;
+    private Date birthday;
+    private Collection<TicketSO> tickets;
+
+
+    public int getId() {
+        return id;
+    }
+    public void setId(int id) {
+        this.id = id;
+    }
+
+
+    public String getFirstName() {
+        return firstName;
+    }
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public Date getBirthday() {
+        return birthday;
+    }
+    public void setBirthday(Date birthday) {
+        this.birthday = birthday;
+    }
+
+    public Collection<TicketSO> getTickets() {
+        return tickets;
+    }
+    public void setTickets(Collection<TicketSO> tickets) {
+        this.tickets = tickets;
+    }
+
+    //TODO
```

Comment:
please provide author how made this TODO and short description what are you mean

Reply:
TODO: add equals & hashCode
Done.